byterun/Makefile.nt

```diff
@@ -44,11 +45,11 @@ libcamlrund.$(A): $(DOBJS)
 	$(call MKLIB,libcamlrund.$(A),$(DOBJS))
 
 %.$(O): %.c
-	$(CC) $(CFLAGS) $(BYTECCCOMPOPTS) -c $<
+	$(CC) $(CFLAGS) -c $<
 
 # It is imperative that there is no space after $(NAME_OBJ_FLAG)
 %.$(DBGO): %.c
-	$(CC) $(DFLAGS) $(BYTECCDBGCOMPOPTS) -c $(NAME_OBJ_FLAG)$@ $<
+	$(CC) $(DFLAGS) -c $(NAME_OBJ_FLAG)$@ $<
```

Comment:
Is $(BYTECCDBGCOMPOPTS) the same as $(BYTECCCOMPOPTS)? If not, some difference has been introduced here.